packages/eslint-config-shopify/rules/stylistic-issues.js

```diff
@@ -23,6 +23,8 @@ module.exports = {
   'func-names': 0,
   // Enforces use of function declarations or expressions
   'func-style': [1, 'declaration'],
+  // Blacklist certain identifiers to prevent them being used
+  'id-blacklist': [1, 'e'],
```

Comment:
Not sure, haha. No point in doing it anyway, since it's handled by the `id-length` rule.